chassis/configuration/configuration/reporting/session-comparison-report.jrxml

```diff
@@ -29,8 +29,9 @@
     <property name="ireport.y" value="0"/>
     <import value="com.griddynamics.jagger.engine.e1.sessioncomparation.SessionVerdict"/>
     <template class="java.io.InputStream">
-        <![CDATA[$P{context}.getResource("configuration/reporting/styles/default-styles.xml")]]></template>
+        <![CDATA[$P{context}.getResource("/reporting/styles/default-styles.xml")]]></template>
```

Comment:
Because this path only valid for Jagger archetypes and is not valid once configuration jar is in classpath